ENIGMAsystem/SHELL/Platforms/SDL/Joystick.cpp

```diff
@@ -64,7 +64,7 @@ bool joystick_button(int id, int numb) {
 double joystick_axis(int id, int numb) {
   double axis;
   double pos = 0;
-  axis = SDL_JoystickGetAxis(joystick_get_handle(id), axis - 1);
+  axis = SDL_JoystickGetAxis(joystick_get_handle(id), numb - 1);
   if (axis != -256 && axis != 0) // don't ask why this shit is necessary; it just is...
```

Comment:
@time-killer-games what was the point of this condition? This function is supposed to return between the closed interval of [-1,1] and this seems to create a hole in said interval.

Reply:
I was drawing the return values and it would either return -256 or zero depending on weird conditions and it's been a while so I can't remember under what conditions it did that.

Reply:
I'm not convinced then that the condition should be kept the way it is. What you were experiencing is likely the deadzone of the joystick axis. In fact, the `pos = -0.01;` branch is probably never even being reached like you think it is. I'll have to investigate later in some future pr.